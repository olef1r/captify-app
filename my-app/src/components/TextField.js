import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import TextField from '@material-ui/core/TextField';
import Container from '@material-ui/core/Container';

const useStyles = makeStyles(() => ({
  textField: {
    marginLeft: '44rem',
    width: 200,
  },
}));

export default function BasicTextFields({ onChangeValue, value }) {
  const classes = useStyles();
  return (
    <Container maxWidth="xs">
      <form className={classes.container} noValidate autoComplete="off">
          <TextField
            id="standard-basic"
            className={classes.textField}
            label="Enter item"
            margin="normal"
            onChange={onChangeValue}
            value={value}
          />
      </form>
    </Container>
  );
}
