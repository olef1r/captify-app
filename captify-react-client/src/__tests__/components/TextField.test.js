import React from 'react';
import { shallow } from 'enzyme';
import TextField from '../../components/TextField';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<TextField/>);
});

describe('<TextField - Component', () => {
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toContainExactlyOneMatchingElement('form');
  });
});
