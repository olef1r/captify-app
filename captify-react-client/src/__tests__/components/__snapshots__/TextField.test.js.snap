// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField - Component renders without crashing 1`] = `
<WithStyles(ForwardRef(Container))
  maxWidth="xs"
>
  <form
    autoComplete="off"
    noValidate={true}
  >
    <WithStyles(ForwardRef(TextField))
      className="makeStyles-textField-1"
      id="standard-basic"
      label="Enter item"
      margin="normal"
    />
  </form>
</WithStyles(ForwardRef(Container))>
`;
